js/components/adOverlay.js

```diff
@@ -1,10 +1,10 @@
-/********************************************************************
+/** ******************************************************************
   AD OVERLAY
 *********************************************************************/
 var React = require('react'),
-    ClassNames = require('classnames'),
-    CloseButton = require('./closeButton'),
-    CONSTANTS = require('../constants/constants');
+  ClassNames = require('classnames'),
```

Comment:
is there rule for enforcing that you must use 'var' for new variables? @aeng7  what do you think?

Reply:
@pilievOoyala  yeah there's https://eslint.org/docs/rules/one-var if I understand you correctly